src/Compilers/Core/Portable/Symbols/ITypeSymbol.cs

```diff
@@ -94,4 +94,14 @@ public interface ITypeSymbol : INamespaceOrTypeSymbol
         /// </param>
         ISymbol FindImplementationForInterfaceMember(ISymbol interfaceMember);
     }
+
+    static internal class ITypeSymbolHelpers
+    {
+        // Intentionally not an extension method. We don't want it ever be called for symbol classes
+        // Once Default Interface Implementations are supported, we can move this method into the interface. 
```

Comment:
:bulb: if this matters, file it as an issue on dotnet/roslyn. Either way this comment seems better if removed.